lib/css-base.js

```diff
@@ -54,7 +54,7 @@ function cssWithMappingToString(item, useSourceMap) {
 		return content;
 	}
 
-	if (useSourceMap) {
+	if (useSourceMap && typeof btoa === 'function') {
```

Comment:
BREAKING CHANGE for node <= 14!
Extracting of source map SHOULD NOT be ignored!
Use the `Buffer.from(string).toString('base64')`

Reply:
This code for browser